### DISCUSSION TITLE
Can't change bullet colors

### DISCUSSION THREAD

**Discussion by Taliesin1** [SlRvb/Obsidian--ITS-Theme#221] [Q&A]
I'm trying to change the bullet colors in Colors > Note Elements > Lists, and nothing happens. Where am I going wrong?

**Comment by SlRvb** [comment]
I'm not sure what's happening there, testing on ITS v1.0.56 that settings seems to change the color correctly. Only thing I can think of is not changing the color for the correct mode (dark/light).

**Comment by Taliesin1** [comment]
Well, I'm in dark mode for sure. Here's a screenshot of the control panel where you can see I've changed the bullets to green, but my popped-out note is still showing red bullets.

<img width="1317" alt="Screenshot 2023-07-03 at 9 50 18 PM" src="https://github.com/SlRvb/Obsidian--ITS-Theme/assets/10515719/3e298abe-d02d-4ac9-a238-ccbd05600162">
